src/config_objects.c

```diff
@@ -219,6 +224,14 @@ TbBool parse_objects_object_blocks(char *buf, long len, const char *config_textn
     arr_size = object_conf.object_types_count;
     for (i=0; i < arr_size; i++)
     {
+        short sprite_max_size = 0;
+        int sprite_x = 0, sprite_y = 0, sprite_w = 0, sprite_h = 0;
+        int sprite_x_3d = 0, sprite_y_3d = 0, sprite_w_3d = 0, sprite_h_3d = 0;
+        char anim_path[COMMAND_WORD_LEN];
+        char anim_path_3d[COMMAND_WORD_LEN];
```

Comment:
Would be nice to change '3d' in the code to 'fp' too. Here and in other locations.

Reply:
obsolete